dev-python/flask-api/flask-api-3.1-r1.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( pypy3 python3_{10..12} )
+
+inherit distutils-r1
+
+DESCRIPTION="Browsable web APIs for Flask"
+HOMEPAGE="
+	https://github.com/flask-api/flask-api/
+	https://pypi.org/project/Flask-API/
+"
+# pypi mirror don't have docs folder
+SRC_URI="
+	https://github.com/flask-api/flask-api/archive/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	dev-python/flask[${PYTHON_USEDEP}]
```

Comment:
Since flask-3 requires werkzeug-3, shouldn't it also be restricted?

Reply:
Won't portage cover that transitively?  Are packages supposed to know about dependencies of dependencies?

Reply:
It's a confusing mess to require PM to figure that out. I doubt flask-3 works anyway.

Reply:
Sure, done.